packages/dbos-cloud/applications/delete-app.ts

```diff
@@ -10,6 +10,10 @@ export async function deleteApp(host: string): Promise<number> {
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const packageJson = require(path.join(process.cwd(), 'package.json')) as { name: string };
   const appName = packageJson.name;
+  if (appName === undefined) {
+    logger.error("Error: package.json not found. Please run this command in an application root directory.")
```

Comment:
Maybe say `Error: cannot find a valid package.json file. Please run this command in an application root directory`

Reply:
Done!